migrations/20190329111919_add-locations.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function (knex) {
```

Comment:
Can you point me to where we're getting all the info to go into this table, like `city`, `location`, etc? From the code, I only saw data coming from the Athena query which isn't getting this data, unless I missed it.